kratos/utilities/exact_mortar_segmentation_utility.h

```diff
@@ -500,87 +513,143 @@ class KRATOS_API(KRATOS_CORE) ExactMortarIntegrationUtility
 
     /**
      * @brief This function push backs the points that are inside
-     * @param PointList The intersection points
-     * @param AllInside The nodes that are already known as inside the other geometry
-     * @param ThisGeometry The geometry considered
+     * @param rPointList The intersection points
+     * @param rAllInside The nodes that are already known as inside the other geometry
+     * @param rThisGeometry The geometry considered
      */
+    template<SizeType TSizeCheck = TNumNodes>
```

Comment:
you just moved this function from the cpp bcs it is templated now right?

Reply:
Yes, many errors, it was easier to move this